src/app/ui-kit/typography/typography.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-typography',
```

Comment:
let's add ui or ui-kit in selector name to mark out ui-kit components (for example, app-ui-typography)